tools/data_export/export_remote.xml

```diff
@@ -0,0 +1,55 @@
+<tool id="export_remote" name="Export datasets" version="0.1.0" profile="21.05">
+    <description>to remote files source</description>
+    <command><![CDATA[
+        python '$__tool_directory__/export_remote.py'
+            --file-sources '$file_sources'
+            --directory-uri '$d_uri'
+            #if $export_type.export_type_selector == "datasets_auto"
+                --infiles
+                #for $filepath in $export_type.infiles:
+                    '$filepath'
+                #end for
+                --names
+                #for $infile in $export_type.infiles:
+                    '$infile.name_and_ext'
```

Comment:
- [ ] We should also build a variant that is file system safe
- [ ] We should move this into a configfile to avoid shell limits
- [ ] We should export MetadataFile objects as well (maybe configurable ?)